### PR TITLE
publish-iis command

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: src/Microsoft.AspNet.Tools.PublishIIS/Program.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                var appName = appNameOption.Value();
+
+                if (publishFolder == null || appName == null)
+                {
+                    app.ShowHelp();
+                    return 2;
+                }
+
+                XDocument webConfigXml = null;
+                var webConfigPath = Path.Combine(publishFolder, "wwwroot", "web.config");
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Tools.PublishIIS/Program.cs]: wwwroot can be renamed via the webroot parameter in hosting.json or command line.

File: src/Microsoft.AspNet.Tools.PublishIIS/project.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿{
+  "name": "publish-iis",
+  "version": "1.0.0-*",
+  "compilationOptions": {
+      "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "NETStandard.Library": "1.0.0-rc2-23616",
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Tools.PublishIIS/project.json]: fixed versions?

File: src/Microsoft.AspNet.Tools.PublishIIS/Program.cs (hunk 1 above)
Comment (moozzyk) [inline src/Microsoft.AspNet.Tools.PublishIIS/Program.cs]: Yes, I am planning to handle this separately.

**Comment by Tratcher** [comment]
Please add a test case to make sure this bug in the prior implementation has been fixed: https://github.com/aspnet/dnx/issues/3062

File: src/Microsoft.AspNet.Tools.PublishIIS/project.json (hunk 2 above)
Comment (moozzyk) [inline src/Microsoft.AspNet.Tools.PublishIIS/project.json]: Will fix this

**Comment by moozzyk** [comment]
OK

**Comment by moozzyk** [comment]
:up::date:
- unpinned package versions i project.json
- Moved from CommandLineUtils.Sources to CommandLineUtils
- Added test to verify that children of httpPlatform element are not removed (aspnet/dnx#3602)

File: src/Microsoft.AspNet.Tools.PublishIIS/Program.cs (hunk 1 above)
Comment (Tratcher) [inline src/Microsoft.AspNet.Tools.PublishIIS/Program.cs]: Re: https://github.com/aspnet/Hosting/issues/449

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**